packages/ast-spec/package.json

```diff
@@ -34,7 +34,7 @@
     "postclean": "rimraf dist && rimraf _ts3.4 && rimraf .rollup.cache && rimraf coverage",
     "clean-fixtures": "rimraf -g \"./src/**/fixtures/**/snapshots\"",
     "format": "prettier --write \"./**/*.{ts,mts,cts,tsx,js,mjs,cjs,jsx,json,md,css}\" --ignore-path ../../.prettierignore",
-    "lint": "eslint . --ignore-path='../../.eslintignore'",
+    "lint": "cd ../../ && nx lint @typescript-eslint/ast-spec",
```

Comment:
I kept these npm script aliases as @bradzacher had previously requested back when I applied this same change to scope-manager.

Personally I run everything from the root _or_ let the Nx Console VSCode extension give me a clickable target runner in the `project.json` of the relevant project:

(You can see it here between lines 5 and 6)

<img width="665" alt="image" src="https://user-images.githubusercontent.com/900523/202895374-902fe3af-30a3-49bb-a863-78d1287d3c0b.png">

Reply:
Nifty, thanks. Is there a way around the `cd ../..`, though? It's kind of annoying to be shoved back into the root directory. Sometimes I do want to have my shell specifically in a package dir.

Reply:
I _think_ there are still some rough edges with running nx from a nested location, but having said that I tried the example of running:

```sh
nx lint @typescript-eslint/ast-spec
```

Within `./packages/ast-spec` and it worked just fine...

It's possible in simple cases like this there are no issues and/or my knowledge is out of date.

I've pushed up an additional commit which removes the cd from all of them 🤞